models/basgra/inst/template.job

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# redirect output
+exec 3>&1
+exec &> "@OUTDIR@/logfile.txt"
+
+# host specific setup
+@HOST_SETUP@
+
+# create output folder
+mkdir -p "@OUTDIR@"
+
+# see if application needs running
+if [ ! -e "@OUTDIR@/results.csv" ]; then
+    
+  "@BINARY@"
+    # convert to MsTMIP
+  echo "require (PEcAn.BASGRA)
+run_BASGRA('@SITE_MET@', @RUN_PARAMS@, '@START_DATE@', '@END_DATE@', '@OUTDIR@', @SITE_LAT@, @SITE_LON@)
+" | R --vanilla
```

Comment:
Not strictly necessary for this PR, but I wonder if this would work as:

```sh
Rscript \
  -e "library(PEcAn.BASGRA)" \
  -e "run_BASGRA('@SITE_MET', @RUN_PARAMS, ...)"
```

`R --vanilla` ignores the `.Renviron` and `.Rprofile` files, which makes it difficult to configure default R library locations, etc.

Reply:
I _would_ strongly consider changing `require` to `library` though -- you do actually want the script to throw an informative error here if `PEcAn.BASGRA` isn't installed.